modules/dxp/apps/osb/osb-faro/osb-faro-web/.eslintrc.js

```diff
@@ -12,7 +12,6 @@ module.exports = {
 	},
 	extends: [
 		'liferay',
-		'plugin:jsx-a11y/recommended',
```

Comment:
Hey @SelenaAungst , this isn't related to the module you are removing. This should stay there.